Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -90,24 +91,6 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-// Always load up to n messages when user arrives.
```

Comment:
There's a lot of red in this diff.  I'd love to continue pulling out other pieces of CVC into separate modules.